src/Fantomas.Core.Tests/ExperimentalDoubleIdentParametersTests.fs

```diff
@@ -0,0 +1,64 @@
+﻿module Fantomas.Core.Tests.ExperimentalDoubleIdentParametersTests
+
+open NUnit.Framework
+open FsUnit
+open Fantomas.Core.Tests.TestHelpers
+
+let config =
+    { config with
+        ExperimentalDoubleIndentParameters = true
+        MaxLineLength = 10 }
+
+[<Test>]
+let ``initial sample`` () =
+    formatSourceString
+        """
+let sillyfuncWithParams parameterName1 ignoredParameterName2 ignoredParameterName3 =
+    longBody
+"""
+        config
+    |> prepend newline
+    |> should
+        equal
+        """
+let sillyfuncWithParams
+        parameterName1
+        ignoredParameterName2
+        ignoredParameterName3
+        =
+    longBody
+"""
+
+[<Test>]
+let ``sample in long member`` () =
+    formatSourceString
+        """
+type X() =
+    member x.Y(a:int, b:int, c:int, d:int, e:int) =
+    
+        // Long body goes here...
+        longBody
+"""
+        config
+    |> prepend newline
+    |> should
+        equal
+        """
+type X() =
+    member x.Y
```

Comment:
@auduchinok this does the double indent but has the multiline tuple parameter.
According to the existing lore, we indent each entry of the tuple.
Does this meet your expectations?